src/Util/Test.php

```diff
@@ -967,7 +981,7 @@ private static function getBooleanAnnotationSetting($className, $methodName, $se
             }
         }
 
-        return null;
+        return;
```

Comment:
I added a new pull request, without this change.
https://github.com/sebastianbergmann/phpunit/pull/2664